src/codegen/compile_classical.ml

```diff
@@ -5110,8 +5115,17 @@ module IC = struct
       E.add_func_import env "ic0" "msg_caller_size" [] [I32Type];
       E.add_func_import env "ic0" "msg_cycles_available128" [I32Type] [];
       E.add_func_import env "ic0" "msg_cycles_refunded128" [I32Type] [];
-      E.add_func_import env "ic0" "msg_cycles_accept128" [I64Type; I64Type; I32Type] [];
+      E.add_func_import env "ic0" "msg_cycles_accept128" [I64Type; I64Type; i] [];
```

Comment:
It would be a nice improvement to retrofit this convention to the other APIs too, so that in the end both `classical_` and `enhanced_` end up equal w.r.t. this section. Of course this is for an (optional) separate PR.

Reply:
Yes that's a good idea, I can update that in a separate PR